### PR TITLE
Sample PR

### PR DESCRIPTION
**Pull request by SSIT-user1**

### CODE REVIEW

File: src/Sample.xoo [hunk 1]
```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```
Comment (SSIT-reviewer1) [inline src/Sample.xoo]: Code Smell Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:3000/coding_rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:3000/project/issues?id=sample&branch=feature1&issues=AWDq3PphOiCDCPFu2Udh&open=AWDq3PphOiCDCPFu2Udh)

[ISSUE_KEY]: AWDq3PphOiCDCPFu2Udh

**Review by ghost — COMMENTED** [review]
foo

**Review by ghost — DISMISSED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
x

**Review by ghost — DISMISSED** [review]

**Review by ghost — COMMENTED** [review]
comment

File: src/Sample.xoo [hunk 2]
```diff
@@ -10,3 +10,5 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
```
Comment (janos-ss) [inline src/Sample.xoo]: bad1

File: src/Sample.xoo [hunk 3]
```diff
@@ -10,3 +10,5 @@ public class Sample1 {
...
   }
```
Comment (janos-ss) [inline src/Sample.xoo]: bad2

File: src/Sample.xoo (hunk 2 above)
Comment (janos-ss) [inline src/Sample.xoo]: bad1

**Review by ghost — CHANGES_REQUESTED** [review]
request changes

**Review by SSIT-reviewer1 — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Pull request closed without merging**